pom.xml

```
@@ -8,17 +8,19 @@
         <relativePath />
     </parent>
     <artifactId>mercurial</artifactId>
-    <version>2.10-SNAPSHOT</version>
+    <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
     <name>Jenkins Mercurial plugin</name>
     <description>Allows Jenkins to check out projects from the Mercurial SCM.</description>
     <url>https://wiki.jenkins.io/display/JENKINS/Mercurial+Plugin</url>
     
     <properties>
-      <jenkins.version>2.150.3</jenkins.version>
-      <java.level>8</java.level>
-      <no-test-jar>false</no-test-jar>
-      <reuseForks>false</reuseForks> <!-- long story -->
+        <revision>2.9</revision>
```

Bad merge conflict resolution:

```suggestion
        <revision>2.10</revision>
```

Tip:

```ini
[merge]
	conflictstyle = diff3
```